src/Consolonia.Core/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Consolonia.Core": {
+      "commandName": "Project",
+      "commandLineArgs": "s:\\github\\consolonia\\src\\consolonia.gallery\\gallery\\galleryviews\\gallerytextlblock.axaml --buffer 80 50"
```

Comment:
Do we need this file for non-executable?

Reply:
oh, no, I think we can delete it.  Sorry.